### ISSUE TITLE
Reduce vertical spacing between rows

### ISSUE THREAD

**Issue by muggleveela** [simonbengtsson/jsPDF-AutoTable#887]
I am not able to reduce the vertical line spacing between table rows (default value is currently 1.15). It remains at 1.15 line spacing.

```
const doc: any = new jsPDF('landscape');
const rows = [];
const cols = ['Amount','Description'];
const title = 'Sales details';

rows.push(...this.getSalesData());
doc.lineHeightProportion = 0.5; 
doc.setLineHeightFactor(0.5);

doc.autoTable(cols, rows, {
      styles: {
        lineHeight: 0.5 // not working 
      },
      head: [
         [{ content: title, colSpan: 6, styles: { halign: 'center', cellPadding: 10 } }],
      ],
      startY: 10,
      theme: 'plain',
      didParseCell: (data) => {
        if (data.section === 'body') {
         data.cell.styles.fontStyle = 'bold';
        }
      }
});
``
![XNgR1](https://user-images.githubusercontent.com/106766278/171684775-22b112c2-a31c-457e-9720-ac6a1db84840.png)
`

**Comment by simonbengtsson** [comment]
You likely have to reduce the font size. There is no option for lineHeight as far as I know.

**Comment by muggleveela** [comment]
Thank you, buy I need my font size fixed to 12 and row line spacing less than 1.5. Will there be any release soon to cater for this issue? Any other package that offers this flexibility please?

**Comment by simonbengtsson** [comment]
No one is working on this as far as a I know, but would be a great addition if you want to look into it!
